### PR TITLE
adding country codes dropdown to modal

### PR DESCRIPTION
**Pull request by jhonny-m**

### CODE REVIEW

File: apps/gp2-frontend/src/users/country-codes-suggestions.ts [hunk 1]
```diff
@@ -0,0 +1,1213 @@
+export const countryCodesSuggestions = [
+  {
+    name: 'Afghanistan',
+    dialCode: '+93',
+    code: 'AF',
```
Comment (markcsims) [inline apps/gp2-frontend/src/users/country-codes-suggestions.ts:5]: Do we need the code?

**Review by markcsims — APPROVED** [review]

File: apps/gp2-frontend/src/users/country-codes-suggestions.ts (hunk 1 above)
Comment (jhonny-m) [reply apps/gp2-frontend/src/users/country-codes-suggestions.ts:5]: we don't use the codes
I can clean them up

File: apps/gp2-frontend/src/users/country-codes-suggestions.ts
Comment (dianaguimaraes) [inline apps/gp2-frontend/src/users/country-codes-suggestions.ts:1]: should this be moved to the components instead and avoid passing it on all `ContactInformationModal` components which seems to be what we're doing?

File: packages/react-components/src/molecules/LabeledDropdown.tsx [hunk 2]
```diff
@@ -52,7 +56,9 @@ export default function LabeledDropdown<V extends string>({
             </span>
           )}
           <br />
```
Comment (dianaguimaraes) [inline packages/react-components/src/molecules/LabeledDropdown.tsx:58]: do we want the `br` even if there is no description?

File: apps/gp2-frontend/src/users/country-codes-suggestions.ts
Comment (jhonny-m) [reply apps/gp2-frontend/src/users/country-codes-suggestions.ts:1]: I just kept the behaviour of what has been done previously

**Pull request merged**